### DISCUSSION TITLE
Where is glob syntax documented?

### DISCUSSION THREAD

**Discussion by vatosarmat** [sharkdp/fd#1353] [Q&A] [answered]
Where is glob syntax documented?

**Comment by tmccombs** [comment, marked as answer]
It uses the syntax from this crate: https://docs.rs/globset/latest/globset/#syntax

but it might be good to add that to the man page.

**Reply by tmccombs** [reply]
oh, actually that _is_ mentioned in the man page, under "PATTERN SYNTAX"
